salt/_logging/impl.py

```diff
@@ -121,9 +121,9 @@ class __StoreLoggingHandler(TemporaryLoggingHandler):
 class SaltLogRecord(logging.LogRecord):
     def __init__(self, *args, **kwargs):
         logging.LogRecord.__init__(self, *args, **kwargs)
-        self.bracketname = '[{:<17}]'.format(self.name)
-        self.bracketlevel = '[{:<8}]'.format(self.levelname)
-        self.bracketprocess = '[{:>5}]'.format(self.process)
+        self.bracketname = '[{:<17}]'.format(str(self.name))
+        self.bracketlevel = '[{:<8}]'.format(str(self.levelname))
+        self.bracketprocess = '[{:>5}]'.format(str(self.process))
```

Comment:
Isn't casting the args to `format` to a string redundant?

Reply:
Yes, but when python creates a new log record from a dictionary it passes `None` initially and format blows up with that.
It then updates the classes `__dict__` with valid data.
This cast is just to avoid that traceback.

And kudos for the review.